### ISSUE TITLE
Availability -> Disponibilidade

### ISSUE THREAD

**Issue by rffontenelle** [python/python-docs-pt-br#251]
Undo https://github.com/python/python-docs-pt-br/commit/6510df2a43dda5afe49c953c6d3c1feeb7bd3d7e (retranslate Availability to Disponibilidade) when https://github.com/python/cpython/issues/126609 is solved.

**Comment by rffontenelle** [comment]
Patch on main, waiting the backport https://github.com/python/cpython/pull/129584

**Comment by m-aciek** [comment]
The backport is now merged

**Comment by m-aciek** [comment]
Did we update the template/source strings? I still don't see `Availability` in `sphinx` resource. Isn't the upload part of Friday CI? The merge in 3.13 happened after the Friday CI run.

**Comment by rffontenelle** [comment]
You're right. After updating the source strings and translating sphinx's "Availability", the error stopped. I manually triggered an workflow run to end this now.

**Comment by StanFromIreland** [comment]
Yep I reverted them to the translated version!
